_data/engine-cli/docker_build.yaml

```diff
@@ -172,8 +169,7 @@ options:
   default_value: '[]'
   description: Security options
 - option: shm-size
-  default_value: "0"
-  description: Size of /dev/shm
+  description: Size of /dev/shm, default value is 64MB
```

Comment:
Not sure if it applies in this particular case, but many defaults are set daemon side, setting them by the client would lead to the daemon not able to apply defaults. It's a known issue that we cannot show those defaults in the client (no real solution yet, other than, e.g. requesting the defaults from the daemon on each `--help` / `usage` print 😞